ios-i-book-lover/Data/Model/Author/Author.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Author.swift
+//  BookLover
+//
+//  Created by nguyen.nam.khanh on 4/7/19.
+//  Copyright © 2019 nguyen.nam.khanh. All rights reserved.
+//
+
+import Foundation
+import XMLMapper
+
+final class Author: XMLMappable {
+    var nodeName: String! = ""
```

Comment:
protocol 'XMLBaseMappable'
```suggestion
    var nodeName: String!
```